building-srpm-in-copr/README.md

```diff
@@ -0,0 +1,92 @@
+Copr provides an option to use their [custom source method](https://docs.pagure.org/copr.copr/custom_source_method.html)
+to create sources for SRPM.
+
+## How to use the custom source method to create sources
+
+- required script in any scripting language and chroot where the script is executed (defaults to `fedora-latest-x86_64`)
+- output of the script: specfile, optionally any other file needed to successfully build a source RPM from that spec file (tarballs, patches)
+- script is executed under non-privileged user -> packages needed for running script should be specified as a list of (srpm)build-dependencies
+- optional parameters:
+  - builddeps: list of packages
+  - resultdir: dir with output files (defaults to current working directory)
+- cannot be run separately without creating a copr build
+  (we cannot use it for koji builds)
```

Comment:
We could request this functionality be implemented in copr - though hard to say if the copr team would find this useful and how long it would take to implement